src/core.js

```diff
@@ -329,7 +329,9 @@ Resolved.prototype.toString = function Resolved$toString(){
 
 export const of = x => new Resolved(x);
 
-function Never(){}
+function Never(){
+  this._isNever = true;
+}
```

Comment:
The reason I'm setting it in the constructor, rather than on the prototype, is that it will cause `inspect` to provide slightly more insightful input (`Future { _isNever: true }` vs `Future {}`).